spp_area/models/area_import.py

```diff
@@ -142,7 +148,7 @@ def import_data(self):  # noqa: C901
 
                                 # Determine the column name based on First Row rowIndex(0), columnIndex(col)
                                 col_name = sheet.cell(0, col).value
-                                if col_name.find(str(xcols) + "Pcode") >= 0:
+                                if col_name.find(str(xcols) + "Code") >= 0:
```

Comment:
Why change this? I think it break the compatibility

Reply:
I have fixed the compatibility now that it will work in both templates.